applications/tari_dan_wallet_cli/src/command/transaction.rs

```diff
@@ -135,6 +136,20 @@ pub struct ClaimBurnArgs {
     common: CommonSubmitArgs,
 }
 
+#[derive(Debug, Args, Clone)]
+pub struct SendArgs {
+    #[clap(long, short = 's')]
+    source_address: String,
+    #[clap(long, short = 'w')]
+    amount: u128,
+    #[clap(long, short = 'a')]
+    resource_address: ResourceAddress,
+    #[clap(long, short = 'd')]
+    dest_address: ComponentAddress,
+    #[clap(flatten)]
+    common: CommonSubmitArgs,
```

Comment:
In order to make the command `send <account_name> <amount> <resource_addr> <dest_account>`
remove the clap flag attributes:

Also the account_address should be the account name

```suggestion
    source_account_name: String,
    amount: u128,
    resource_address: ResourceAddress,
    dest_address: ComponentAddress,
    #[clap(flatten)]
    common: CommonSubmitArgs,
```

```
$ cargo run --bin tari_dan_wallet_cli --  transactions send --help
USAGE:
    tari_dan_wallet_cli transactions send [OPTIONS] <SOURCE_ADDRESS> <AMOUNT> <RESOURCE_ADDRESS> <DEST_ADDRESS>

ARGS:
    <SOURCE_ACCOUNT_NAME>      
    <AMOUNT>              
    <RESOURCE_ADDRESS>    
    <DEST_ADDRESS>        

OPTIONS:
    -a, --account-template-address <ACCOUNT_TEMPLATE_ADDRESS>              
    -d, --dump-outputs-into <DUMP_OUTPUTS_INTO>                            
        --dry-run                                                          
        --fee <FEE>                                                        
    -h, --help                                                             Print help information
    -i, --inputs <INPUTS>                                                  
    -m, --non-fungible-mint-outputs <NON_FUNGIBLE_MINT_OUTPUTS>            
    -n, --num-outputs <NUM_OUTPUTS>                                        
        --new-address-list-item-outputs <NEW_ADDRESS_LIST_ITEM_OUTPUTS>    
        --new-non-fungible-outputs <NEW_NON_FUNGIBLE_OUTPUTS>              
    -o, --override-inputs <OVERRIDE_INPUTS>                                
    -t, --wait-for-result-timeout-secs <WAIT_FOR_RESULT_TIMEOUT_SECS>      Timeout in seconds
    -v, --version <VERSION>                                                
```

Reply:
Pretty cool ! Refactor it accordingly :)